app/controllers/admin/content/system-images.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import { eventTopics } from 'open-event-frontend/utils/dictionary/event';
+import { keys, uniqBy } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  imageTopic: keys(eventTopics)[0],
+
+  topics: computed(function() {
+    return keys(eventTopics);
+  }),
+
+  imageObjects: computed('imageTopic', function() {
+    let subTopics = eventTopics[this.get('imageTopic')];
+    let filteredImageArray = this.get('model').map(function(obj) {
```

Comment:
Mmmm ... the Other causes us another problem as we don't know which topic the other belongs too. Hold on. We're changing backend implementation.